extensions/integration-tests/src/notebook.test.ts

```diff
@@ -6,22 +6,39 @@
 'use strict';
 
 import 'mocha';
-import assert = require('assert');
-
+import * as assert from 'assert';
 import * as azdata from 'azdata';
 import * as vscode from 'vscode';
 import { context } from './testContext';
-import { sqlNotebookContent, writeNotebookToFile, sqlKernelMetadata } from './notebook.util';
+import { sqlNotebookContent, writeNotebookToFile, sqlKernelMetadata, getFileName, pySparkNotebookContent, pySpark3KernelMetadata, pythonKernelMetadata } from './notebook.util';
 import { getBdcServer } from './testConfig';
 import { connectToServer } from './utils';
 import * as fs from 'fs';
 
 if (context.RunTest) {
-	suite('Notebook integration test suite', function() {
+	suite('Notebook integration test suite', function () {
+		setup(function () {
+			console.log(`Start "${this.currentTest.title}"`);
+		})
+		teardown(function () {
+			let testName = this.currentTest.title;
+			try {
+				let fileName = getFileName(testName);
+				if (fs.existsSync(fileName)) {
```

Comment:
Since you're just catching and ignoring the error anyways not much point in checking that the file exists. (and this is a common anti-pattern anyways since nothing is preventing the file from being deleted between when you check if it exists and when you try to actually delete it)